packages/typespec-vscode/test/extension/common/commonSteps.ts

```diff
@@ -0,0 +1,355 @@
+import { Locator, Page } from "playwright"
```

Comment:
please fix the ci failues, there is formatting, linting and changelog missing

Reply:
make sure you have installed the right vscode extension so those errors get shown and fixed in the ide